Find a functionally equivalent code:com.android.keyguard.FaceUnlock.handleServiceConnected:COMMENT
Method Modifier: 
Comment:/**
 * Tells the service to start its UI via an AIDL interface.  Called when the
 * onServiceConnected() callback is received.
 */

@@ -20,26 +20,26 @@
             // When switching between portrait and landscape view while Face Unlock is running,
             // the screen will eventually go dark unless we poke the wakelock when Face Unlock
             // is restarted.
-            mKeyguardScreenCallback.userActivity(0);
+            mKeyguardScreenCallback.userActivity();
             int[] position;
             position = new int[2];
             mFaceUnlockView.getLocationInWindow(position);

